doc/quickstart/configure.rst

```diff
@@ -479,58 +470,157 @@ speed with which the data can be read from and/or written to disk.
 
 .. _config-dask-default-scheduler:
 
-Dask default scheduler configuration
-------------------------------------
+Custom Dask default scheduler configuration
+-------------------------------------------
 
 The Dask default scheduler can be a good choice for recipes using a small
 amount of data or when running a recipe where not all preprocessor functions
-are lazy yet (see :issue:`674` for the current status). To use the the Dask
-default scheduler, comment out or remove all content of ``~/.esmvaltool/dask.yml``.
+are lazy yet (see :issue:`674` for the current status).
 
 To avoid running out of memory, it is important to set the number of workers
-(threads) used by Dask to run its computations to a reasonable number. By
-default the number of CPU cores in the machine will be used, but this may be
-too many on shared machines or laptops with a large number of CPU cores
+(threads) used by Dask to run its computations to a reasonable number.
+By default, the number of CPU cores in the machine will be used, but this may
+be too many on shared machines or laptops with a large number of CPU cores
 compared to the amount of memory they have available.
 
-Typically, Dask requires about 2GB of RAM per worker, but this may be more
+Typically, Dask requires about 2 GiB of RAM per worker, but this may be more
 depending on the computation.
 
-To set the number of workers used by the Dask default scheduler, create a file
-called ``~/.config/dask/dask.yml`` and add the following
-content:
+To set the number of workers used by the (thread-based) Dask default scheduler,
+use the following configuration:
 
 .. code:: yaml
 
-  scheduler: threads
-  num_workers: 4  # this example sets the number of workers to 4
+  dask:
+    use: local_threaded  # This can be omitted
+    profiles:
+      local_threaded:
+        num_workers: 4
 
+To switch to a process-based scheduler, use:
 
-Note that the file name is arbitrary, only the directory it is in matters, as
-explained in more detail
-`here <https://docs.dask.org/en/stable/configuration.html#specify-configuration>`__.
-See the `Dask documentation <https://docs.dask.org/en/latest/scheduling.html#configuration>`__
-for more information.
+.. code:: yaml
+
+  dask:
+    use: local_processes
+    profiles:
+      local_processes:
+        num_workers: 4
+        scheduler: processes
```

Comment:
This scheduler is [not supported by Iris](https://github.com/SciTools/iris/blob/ec447312cbd05b1a599118efa5d829e5dfaeb3e1/lib/iris/fileformats/netcdf/_dask_locks.py#L46), so I would leave this out.

Reply:
Removed in https://github.com/ESMValGroup/ESMValCore/pull/2616/commits/28d2ee09a2e73aadf55e0dd64c6892c2cf422beb